src/config/common.json

```
@@ -1,12 +1,12 @@
 {
   "APP_NAME": "NFT Claim App",
-  "DEVELOPMENT_KEYRING": true,
   "RPC": {},
   "THEME": "polkadotnft",
   "MAX_AMOUNT": 100,
   "CLAIM_URL": "https://claimnft.polkadot.network/#/claim",
   "ALTERNATIVE_APP_URL": "https://claimnft.kusama.network",
   "NFT_PREVIEW_URL": "https://singular.rmrk.app/collectibles/statemint/",
   "ADDED_FEE_MULTIPLIER": 2,
-  "CHAIN_SCANNER_URL": "https://statemint.subscan.io/"
+  "CHAIN_SCANNER_URL": "https://statemint.subscan.io/",
+  "RELAYCHAIN_GENESIS": "0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3"
```

Set Relaychain genesis to kusama genesis.

```suggestion
  "RELAYCHAIN_GENESIS": "0xb0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe"
```